fedimint-cli/src/lib.rs

```diff
@@ -694,6 +733,83 @@ impl FedimintCli {
                         .map_err_cli_msg("invalid response")?,
                 ))
             }
+            Command::Admin(AdminCmd::WsStatus) => {
```

Comment:
Thought: We should split handling admin commands to a separate function. The whole thing is getting way too long.

Reply:
Ok, reorganized a bit now it's a subcommand and calls another function.